src/providers/twitch/eventsub/Connection.cpp

```diff
@@ -23,15 +25,41 @@ namespace {
 using namespace chatterino;
 using namespace chatterino::eventsub;
 
+namespace channel_moderate = lib::payload::channel_moderate::v2;
+
 // NOLINTNEXTLINE(cppcoreguidelines-avoid-non-const-global-variables)
 const auto &LOG = chatterinoTwitchEventSub;
 
+void handleModerateMessage(
+    TwitchChannel *chan, [[maybe_unused]] const QDateTime &time,
+    const channel_moderate::Event &event,
+    [[maybe_unused]] const channel_moderate::Clear &action)
```

Comment:
Could be done in something like `providers/twitch/eventsub/MessageHandlers.{c,h}pp`?

Reply:
That file structure seems fine to me :+1: